cmd/fleetctl/gitops.go

```diff
@@ -291,6 +347,70 @@ func gitopsCommand() *cli.Command {
 	}
 }
 
+// Merge sets of label names.
+func concatLabels(labelArrays ...[]string) []string {
+	var result []string
+	for _, arr := range labelArrays {
+		result = append(result, arr...)
+	}
+	return result
+}
+
+// Given a set of referenced labels and info about who is using them, update a provided usage map.
+func updateLabelUsage(labels []string, ident string, usageType string, currentUsage *map[string][]LabelUsage) {
```

Comment:
I don't think you need to pass the map in by reference. Confusingly go maps are reference types under the hood

Reply:
That will also help declutter the code where you have to dereference the maps, since it will be unnecessary

Reply:
https://medium.com/@sajo02/pass-by-reference-in-go-maps-and-slices-problems-and-solutions-with-examples-89de1fe1dbca

Reply:
good call, thanks!  updated.